balloon/src/main/java/com/skydoves/balloon/Balloon.kt

```diff
@@ -1230,10 +1230,7 @@ class Balloon(
         (displayWidth * builder.widthRatio).toInt() - spaces
       builder.minWidthRatio != NO_Float_VALUE || builder.maxWidthRatio != NO_Float_VALUE -> {
```

Comment:
We can remove the `minWidthRatio` related conditional like the below.
```kotlin
builder.maxWidthRatio != NO_Float_VALUE -> ...
```

Reply:
The problem with not including `minWidthRatio` is that we fall back to `maxWidth`. Then if we have set `maxWidth` we get the behavior below. If we expect the user to only combine `minWidthRatio` with  `maxWidthRatio` this shouldn't be a problem but otherwise we get an unwanted behavior.
![image](https://user-images.githubusercontent.com/36191409/128673324-02c55670-b411-416b-9ddc-3516b187e1a3.png)
This is with minRatio set to 0.8f and maxWidth set to 200dp